bpf/process/pfilter.h

```diff
@@ -342,15 +342,19 @@ selector_match(__u32 *f, struct selector_filter *sel,
 four:
 	res4 = process_filter(sel, f, enter, &msg->ns, &msg->caps);
 	index = next_pid_value(index, f, ty);
+	sel->index = index;
 three:
```

Comment:
could we do directly `sel->index = next_pid_value(index, f, ty);`  ?

Reply:
maybe not. If we do that, index is not changed. and sel->index will be 0 and 4(using first two pids), not more pids will be choosen.